src/toil_scripts/tools/preprocessing.py

```diff
@@ -34,7 +37,8 @@ def run_cutadapt(job, r1_id, r2_id, fwd_3pr_adapter, rev_3pr_adapter):
         parameters.extend(['-o', '/data/R1_cutadapt.fastq', '/data/R1.fastq'])
     # Call: CutAdapt
     docker_call(tool='quay.io/ucsc_cgl/cutadapt:1.9--6bd44edd2b8f8f17e25c5a268fedaab65fa851d2',
-                work_dir=work_dir, parameters=parameters)
+                work_dir=work_dir,
```

Comment:
I would prefer not to touch this code in this PR, since it is functionally unrelated to the rest of the PR.